### PR TITLE
fix: fix NPE error if project name or project id is null

### PR DESCRIPTION
**Pull request by arvgord**
- Projects with null name or id will be skipped
- Added unit tests

 #37

### CODE REVIEW

File: src/main/java/org/weasis/dicom/google/api/GoogleAPIClient.java [hunk 1]
```diff
@@ -252,7 +252,11 @@ private static void tokenInfo(String accessToken) throws IOException {
                 continue;
             }
             for (Project project : response.getProjects()) {
-                result.add(new org.weasis.dicom.google.api.model.ProjectDescriptor(project.getName(), project.getProjectId()));
+                String name = project.getName();
+                String projectId = project.getProjectId();
+                if (name != null && projectId != null) {
+                    result.add(new org.weasis.dicom.google.api.model.ProjectDescriptor(name, projectId));
```
Comment (poojavenkatram) [inline src/main/java/org/weasis/dicom/google/api/GoogleAPIClient.java:258]: Just curious, what is the implication of this?

Comment (arvgord) [reply src/main/java/org/weasis/dicom/google/api/GoogleAPIClient.java:258]: This means that if the application receives projects from a server without name and projectId field in a response(null), they will be skipped. This prevents NullPointerException.

Comment (poojavenkatram) [reply src/main/java/org/weasis/dicom/google/api/GoogleAPIClient.java:258]: When you say they will be skipped, the user cannot view the data from those Projects?

Comment (arvgord) [reply src/main/java/org/weasis/dicom/google/api/GoogleAPIClient.java:258]: Yes. Users can't create a project without a name and id. 

изображение

Comment (poojavenkatram) [reply src/main/java/org/weasis/dicom/google/api/GoogleAPIClient.java:258]: I am getting this error without typing/creating any project...Like literally signing it and just clicking anywhere on Weasis screen..

Comment (poojavenkatram) [reply src/main/java/org/weasis/dicom/google/api/GoogleAPIClient.java:258]: Let me try what will happen with a new release with this change...

**Comment by poojavenkatram** [comment]
Trying this to see if it fixes the issue...

**Comment by arvgord** [comment]
Let me know if something goes wrong

**Pull request merged**